test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,26 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        // all null objects
+        assertTrue(Utils.isAnyNull((Object) null));
+        assertTrue(Utils.isAnyNull(null, null, null));
+        assertTrue(Utils.isAnyNull(null, null, null, null));
+
+        // some null objects
+        assertTrue(Utils.isAnyNull(1, null, "abc"));
+        assertTrue(Utils.isAnyNull("a", 1, Integer.valueOf(1), null));
+        assertTrue(Utils.isAnyNull(null, 1, 2, "abc", 3, null));
+
+        // no null objects
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull(true));
+        assertFalse(Utils.isAnyNull("abc", 1, false));
+        assertFalse(Utils.isAnyNull());
```

Comment:
Very comprehensive set of tests! However, do consider the case where the object is a list, what if there are null objects within the list?